src/cr/cube/crunch_cube.py

```diff
@@ -473,8 +580,14 @@ def _margin(self, axis=None, weighted=True, adjusted=False,
         res = np.sum(array, axis)
 
         if prune and axis is not None and type(res) is np.ndarray:
-            prune_indices = self.prune_indices(include_transforms_for_dims)
-            res = res[~prune_indices[1 - axis]]
+            table = self.as_array(
+                include_transforms_for_dims=include_transforms_for_dims,
+                prune=True,
```

Comment:
always prune?

Reply:
Yes, if it goes into that if. Fixed anyways, with passing in the actual `prune` argument from the outer scope.